src/service/core-services.js

```diff
@@ -79,7 +79,10 @@ export function installRuntimeServices(global) {
 export function installAmpdocServices(ampdoc, opt_initParams, opt_inabox) {
   // Order is important!
   installUrlForDoc(ampdoc);
-  installCidService(ampdoc);
+  if (!opt_inabox) {
+    // For security, storage service is not supported in inabox.
+    installCidService(ampdoc);
```

Comment:
Does the installation order matter? Can we combine the `opt_inabox` check to one.

Reply:
According to the comment above, the order matters.